### ISSUE TITLE
SFC with external src file errors with Template Interpolation Service enabled

### ISSUE THREAD

**Issue by outofmemoryagain** [vuejs/vetur#1254]
- [X] I have searched through existing issues
- [X] I have read through [docs](https://vuejs.github.io/vetur)
- [X] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: Windows 10
- Vetur version: Vetur 0.19.2
- VS Code version: 
Version: 1.33.1 (system setup)
Commit: 51b0b28134d51361cf996d2f0a1c698247aeabd8
Date: 2019-04-11T08:27:14.102Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

## Problem
Vetur generates an error on a SFC template when the TS script section is in an external file.  This is currently a very common use case for existing TS projects as TS support in SFC was limited until now.  
```
File 'c:/XXXXXX/components/EventDetailsHeader.vue' is not a module.
```
My script section is as follows:
``` javascript
<script lang="ts" src="./EventDetailsHeader.ts"></script>
```

If I move the code from the external file directly to the script section I see the TS Language server error, but after restarting VS Code the error is no longer reported, and the new type features appear to be working correctly

## Reproducible Case

1. In a TS project (Vue CLI 3)
2. Add a new MyComponent.vue component with
``` javascript
<template>
  
</template>

<script lang="ts" src="./MyComponent.ts"/>
```
3. Add a new MyComponent.ts typescript file with
``` javascript
import Vue from "vue";

export default Vue.extend({
  props: {
    
  },
  data() {
   return {
     
   };
  },
  methods: {
    
  }
});
```

If needed I can post a github repo, I'm just tied up at the moment.

**Comment by outofmemoryagain** [comment]
Also, after moving the code from the external file if there are any child components used in that script section that have externally referenced src it will give an error for that files import statement.

**Comment by ktsn** [comment]
Thanks for reporting the issue! I've implemented it via #1255
